packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -23,13 +23,14 @@ function instantiatePrism2(operations: IHttpOperation[]) {
 
 describe('body params validation', () => {
   let server: IPrismHttpServer;
+  let address: string;
 
-  afterAll(() => {
-    return server.fastify.close();
+  afterEach(() => {
+    return server.micri.close();
```

Comment:
The ideal name is actually `server.server` which is also not ideal :/
Since we are using only `.close()` I'd vote on wrapping it and have something like: `server.close()`.

I don't think we should expose the guts of our HTTP server implementation.

Reply:
Yeah, I agree with you. Let's go with that — you can easily copy the signature with TypeScript so we save all the overloads.

Reply:
👍